### ISSUE TITLE
Creating the API endpoint for getting event by id

### ISSUE THREAD

**Issue by KeremZaman** [bounswe/2021SpringGroup2#56]
An API endpoint should be implemented to get events by event IDs.

```/api/v1.0/events/<int:event_id>```

Also, it should show number of current coronavirus cases in the country of event and decide the risk status whether there is a constant increase in cases for 3 consecutive days. covid19api.com will be used for coronavirus stats.

**Comment by KeremZaman** [comment]
I implemented the API endpoint with the additional coronavirus information. Covid19 API allows us to fetch stats only for countries. Since event location does not explicitly contain the country of the location, it was needed to convert location to country. I used GeoNames API, but it supports conversion from names of cities and provinces while event locations can be a name of a sport complex in a street. So, I assumed users create events only in their country and used registration location (usually city of the user) to fetch country.
